data/category-porn

```diff
@@ -2287,13 +2288,16 @@ regexp:(^|\.)fiftymvapi\..+$
 regexp:(^|\.)h5ccav[1-5]\.com$
 regexp:(^|\.)hkfg[1-7]\.xyz$
 regexp:(^|\.)jdyy[1-9]?.me$
+regexp:(^|\.)jrrd([1-9]|1[0-9]|2[0-9]|3[0-9])\.pw$
```

Comment:
Warning: These domains have a relationship with child pornographic. Source from [https://www.youtube.com/watch?v=HzXB-LqoIUc](https://www.youtube.com/watch?v=HzXB-LqoIUc). (An issue in Weibo) I have no idea whether that should be added to this category or not.

Reply:
I personally would like to keep child pornographic content OUT of this project.
The principle for accepting contribution would still need more discussion.
What do you think? @Loyalsoldier @Robot-DaneelOlivaw @EpLiar @kslr @parasomn1a @DuckSoft

Reply:
> I personally would like to keep child pornographic content OUT of this project. 

I agree with that.

Reply:
Until now, Everyone decide to remove child pornographic content. I will remove that content immediately.